resources/fabric.mod.json

```diff
@@ -21,7 +21,7 @@
 		]
 	},
 	"depends": {
-		"fabricloader": ">=0.7.0"
+		"fabricloader": ">=0.12.0"
```

Comment:
ASM9 (beta technically) added `ClassNode#permittedSubclasses` which was first included in Loader 0.10: FabricMC/fabric-loader#308
Whilst I don't think anyone is actually still using a Loader version that old in practice, it doesn't hurt to support the situation all the same.

Reply:
I thought I checked that and found the field to be called `permittedSubclassesExperimental`, but having checked again you seem to be right.